src/styles/getPlatformElevation.ios.js

```diff
@@ -17,9 +18,7 @@ const getPlatformElevation = (elevation) => {
             zIndex: ELEVATION_ZINDEX,
```

Comment:
Also fixes this bug that introduced in https://github.com/xotahal/react-native-material-ui/pull/169